frontend/pages/read_upload.tsx

```diff
@@ -11,6 +11,18 @@ import { SignOutButton } from "components/SignOutButton";
 import { ThemePicker } from "components/ThemePicker";
 import { UserDisplay } from "components/UserDisplay";
 
+const audio_warning = (warning: string) => {

```

Comment:
Functions should be named verbs because they represent an action. A better name for `audio_warning` might be `play_audio_warning`.

Reply:
Also, please use `camelCase` casing where there is not a specific reason to use `snake_case`. `camelCase` is the standard for TypeScript.